common/models/access-token.js

```diff
@@ -143,8 +143,12 @@ module.exports = function(AccessToken) {
       assert(this.ttl >= -1, 'token.ttl must be >= -1');
 
       var AccessToken = this.constructor;
-      var userRelation = AccessToken.relations.user; // may not be set up
-      var User = userRelation && userRelation.modelTo;
+      var User = AccessToken.registry.getModelByType('User');
```

Comment:
I think we need to honour `AccessToken.relations.user.modelTo` as the default `User` model. Consider the case where app has custom User and AccessToken models, e.g. in order to customise some of built-in methods and/or settings.

In that case, `principalType` is not set. Before your change, the code will look up the related user model. After your change, the built-in User model would be incorrectly used.

Could you please add a unit-test for this case?